### PR TITLE
frontend: skip ExcludeArch and ExclusiveArch for resubmitted builds

### LINKED ISSUE #2833: ExclusiveArch / Exclude Arch for resubmitted builds?
https://github.com/fedora-copr/copr/blob/bddee02fb014898075b69e058f1d8fe8a0d13dbf/frontend/coprs_frontend/coprs/views/coprs_ns/coprs_builds.py#L472-L483

We should make sure that we skip those also for resubmitted builds.

### PR DESCRIPTION
**Pull request by FrostyX**
Fix #2833

### CODE REVIEW

File: frontend/coprs_frontend/coprs/logic/builds_logic.py [hunk 1]
```diff
@@ -458,6 +458,13 @@ def create_new_from_other_build(cls, user, copr, source_build,
         build.pkg_version = source_build.pkg_version
         build.resubmitted_from_id = source_build.id
 
+        # Skip previously skipped chroots (ExcludeArch, ExclusiveArch, etc)
+        skipped = [ch.name for ch in source_build.build_chroots
+                   if ch.status == StatusEnum("skipped")]
+        for chroot in build.build_chroots:
+            if chroot.name in skipped:
+                chroot.status = StatusEnum("skipped")
```
Comment (praiskup) [inline frontend/coprs_frontend/coprs/logic/builds_logic.py:466]: Couldn't we place this into backend_enqueue_buildchroots() and have the logic concentrated on one place?

Comment (FrostyX) [reply frontend/coprs_frontend/coprs/logic/builds_logic.py:466]: The `backend_enqueue_buildchroots` method is not called when resubmitting. Should I call it from this place in the code?

Comment (FrostyX) [reply frontend/coprs_frontend/coprs/logic/builds_logic.py:466]: I forgot how the resubmit works. It doesn't use the previous SRPM but rather builds it again. That means the whole python3-specfile parsing happens again and it is send to backend as `results.json` and then sent to frontend as for any other not-resubmitted build. Soooo, #2833 is invalid? When I run frontend from the `main` branch and resubmit package with exclusivearch, it skips the chroots.

Comment (praiskup) [reply frontend/coprs_frontend/coprs/logic/builds_logic.py:466]: Yes, but  *resubnitted srpm ploads*. This is tricky...

**Review by praiskup — APPROVED** [review]
We discussed this 1:1 and we don't have a better (or easier) solution than setting the "skipped" state in two places.  So this is good enough, and better to have than not to have. Thank you!

**Pull request merged**